pkg/projects/Microsoft.NETCore.App/project.json

```diff
@@ -42,7 +42,7 @@
     "Libuv": "1.9.2",
     "Microsoft.NETCore.Platforms": "1.0.2",
     "System.Net.Http": "4.1.3",
-    "runtime.native.System.Security.Cryptography": "4.0.2"
+    "runtime.native.System.Security.Cryptography": "4.0.3"
   },
```

Comment:
Please be sure to check any of the new package versions we are shipping to ensure they are explicitly listed here (assuming they are in Microsoft.NETCore.App).

Reply:
Just validated this and looks like expected. Will go ahead and merge